packages/app-bridge/src/types/Color.ts

```diff
@@ -94,8 +94,17 @@ export type ColorApiPatch = Partial<{
 
 export type Color = {
     id: number;
-    name: Nullable<string>;
     sort: number;
+    color: ColorAsset;
```

Comment:
this `color` property name becomes somewhat redundant now considering that the type itself is already `Color` 🤔 wdyt?

Reply:
True I also wasn't sure, maybe something like `asset` would fit better?

Reply:
not sure if `asset` would be the best alternative as `Color` is already the Asset reference but maybe both `title` and `revision` could be directly referenced here ... is `ColorAsset` really necessary?